example-trail-POST.json

```diff
@@ -0,0 +1,19 @@
+{
+	"data": {
+		"id" : 0,
+		"type" : "Trail",
+	    	"attributes": {
+			"name": "Whoops",
```

Comment:
Not a big deal but a few of these fields are on different indent lines. Feel free to ignore, I'm good either way.